infrastructure/cicsk8s/galasa-prod/galasa-prod/codecov.yaml

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright contributors to the Galasa project
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: codecov
+  namespace: galasa-prod
+  labels:
+    app: codecov
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: codecov
+  template:
+    metadata:
+      name: codecov
+      labels:
+        app: codecov
+    spec:
+      nodeSelector:
+        kubernetes.io/arch: amd64
+        applevel: critical
+      containers:
+      - name: codecov
+        image: docker.galasa.dev/galasa-code-coverage:codecov
```

Comment:
Should this be harbor.galasa.dev (though we probably haven't pushed the code coverage image there yet!)